playbooks/roles/appsembler_reporting/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: make logs directory for appsembler_reporting
+  file:
+    path: "{{ appsembler_reporting_log_dir }}"
+    state: "directory"
+    owner: "{{ edxapp_user }}"
+    group: syslog
+    mode: 0755
+
+- name: Create the reporting scripts directory
+  file:
+    path: "{{ appsembler_reporting_scripts_dir }}"
+    state: directory
+    owner: edxapp
+    group: edxapp    
```

Comment:
Please specify a mode here. I'm thinking of `0775`, but not completely sure about it.